mercurial.html.markdown

```diff
@@ -0,0 +1,387 @@
+---
+category: tool
+tool: hg
+contributors:
+  - ["Will L. Fife", "http://github.com/sarlalian"]
+filename: LearnHG.txt
+---
+
+Mercurial is a free, distributed source control management tool. It offers
+you the power to efficiently handle projects of any size while using an
+intuitive interface. It is easy to use and hard to break, making it ideal for
+anyone working with versioned files.
+
+## Versioning Concepts
+
+### What is version control?
+
+Version control is a system that keeps track fo changes to a set of file(s)
+and/or directorie(s) over time.
+
+
+### Why Use Mercurial
+
+* Distributed Architecture
+* Fast
+* Platform Independent
+* Extensible
+* Easy to use
+* Open Source
+
```

Comment:
Could you merge the two lists into one? The text in the second part can just be integrated in the first list. For example -

```
* Distributed Architecture - Traditionally version control systems such as CVS and Subversion are a client server architecture with a central server to store the revsion history of a project.  Mercurial however is a truly distributed architecture, giving each devloper a full local copy of the entire development history.  It works independently of a central server.
```

Reply:
Keeping in mind line length of 80 chars. I missed that in my comment above.